playbooks/common/openshift-cluster/redeploy-certificates/registry.yml

```diff
@@ -4,99 +4,83 @@
   vars:
   roles:
   - lib_openshift
+  #we need facts as we are including other role tasks
+  - openshift_master_facts
```

Comment:
What is needed from this role?

We should either pull the host-vars from the first master, or fix the scope of whatever variable is needed from openshift_master_facts.

Reply:
Lack of knowledge in the codebase and need to rebase clients code... Will check this when come back on site. It might be few days and heketi is not the bigger priority for the client...